crates/controller/src/mouse.rs

```diff
@@ -209,7 +225,10 @@ fn update_buttons(
     mut mouse_state: ResMut<MouseDragStates>,
     mut input_events: EventReader<MouseButtonInput>,
     mut clicks: EventWriter<MouseClicked>,
+    mut double_clicks: EventWriter<MouseDoubleClicked>,
```

Comment:
Clippy is failing on this function because it takes too many arguments. I just realized that we can keep sending only click events from the system and create another one which listens on the events and creates a DoubleClick if two or more come within a short timespan.